packages/connect-web/src/browserstacktests/buf.gen.yaml

```diff
@@ -0,0 +1,9 @@
+version: v1
+plugins:
+  - name: es
+    out: src/browserstacktests/gen
```

Comment:
How do we generate the eliza service for browserstack in make? 🤔

Reply:
See the `karma:browserstack` script in package.json. It calls the `generate` command as part of the run. I wanted to isolate the generation to browserstack-related commands as much as I could.

Reply:
I think it would be least surprising to have a "generate" script similar to other packages, and to have the buf.gen.yaml at the root. 

Then running `make` will show a diff when the generated code changes, and CI will also pick up the diff (the browserstack workflow doesn't do a diffcheck). It's also easier to port to turborepo (using the same patterns everywhere means we can cache tasks). 

All karma* files are the root are only used for browserstack, so it's not an issue to have buf.gen.yaml there as well.